manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "__MSG_extensionName__",
   "description": "__MSG_extensionDescription__",
-  "version": "2.0.0pre1",
+  "version": "2.0.0pre1+jpy13",
```

Comment:
Could you remove this from the PR?

Reply:
OK, I will push a new commit where this is removed.